browser-test/src/end_to_end_enumerators.test.ts

```diff
@@ -12,46 +12,46 @@ describe('End to end enumerator test', () => {
     const adminPrograms = new AdminPrograms(page);
 
     await adminQuestions.addNameQuestion('enumerator-ete-name');
-    await adminQuestions.addEnumeratorQuestion('enumerator-ete-enumerator', 'desc', 'enumerator-ete-question');
-    await adminQuestions.addNameQuestion('enumerator-ete-enumerator-name', 'desc', 'text', 'helptext', 'enumerator-ete-enumerator');
-    await adminQuestions.addEnumeratorQuestion('enumerator-ete-enumerator-enumerator', 'desc', 'text', 'helptext', 'enumerator-ete-enumerator');
-    await adminQuestions.addTextQuestion('enumerator-ete-enumerator-enumerator-text', 'desc', 'text', 'helptext', 'enumerator-ete-enumerator-enumerator');
+    await adminQuestions.addEnumeratorQuestion('enumerator-ete-householdmembers', 'desc', 'Household members', 'list household members');
+    await adminQuestions.addNameQuestion('enumerator-ete-repeated-name', 'desc', 'Name for \$this', 'full name for \$this', 'enumerator-ete-householdmembers');
+    await adminQuestions.addEnumeratorQuestion('enumerator-ete-repeated-jobs', 'desc', 'Jobs for \$this', '\$this\'s jobs', 'enumerator-ete-householdmembers');
+    await adminQuestions.addNumberQuestion('enumerator-ete-repeated-jobs-income', 'desc', 'Income for \$this.parent\'s job at \$this', 'Monthly income at \$this', 'enumerator-ete-repeated-jobs');

```

Comment:
Question (non-blocking): why revert to `repeated` here and other places?

Reply:
It is because they are repeated questions. Before the questions weren't asking real questions so i just chained the names (e.g. `enumerator-enumerator-text`). Now that the questions are real questions, they have real names. e.g. `repeated-jobs-income` is the repeated question asking about job's income.